.gitignore

```diff
@@ -2,3 +2,26 @@
 *.pyc
 *.glob
 *.aux
+*.meta
+*.o
+*.cmi
+*.cmx
+*.mli
```

Comment:
Out of curiosity, why are you not committing .ml and .mli files, which are any new OCaml files? The rest, though, thank you for